src/cert.rs

```diff
@@ -56,14 +56,20 @@ impl<'a> Cert<'a> {
         cert_der: untrusted::Input<'a>,
         ee_or_ca: EndEntityOrCa<'a>,
     ) -> Result<Self, Error> {
-        let (tbs, signed_data) = cert_der.read_all(Error::BadDer, |cert_der| {
-            der::nested(cert_der, der::Tag::Sequence, Error::BadDer, |der| {
-                // limited to SEQUENCEs of size 2^16 or less.
-                SignedData::from_der(der, der::TWO_BYTE_DER_SIZE)
-            })
-        })?;
+        let (tbs, signed_data) =
+            cert_der.read_all(Error::TrailingData("Cert::wrapper"), |cert_der| {
+                der::nested(
+                    cert_der,
+                    der::Tag::Sequence,
+                    Error::TrailingData("Cert::wrapper"),
+                    |der| {
+                        // limited to SEQUENCEs of size 2^16 or less.
+                        SignedData::from_der(der, der::TWO_BYTE_DER_SIZE)
+                    },
+                )
+            })?;
 
-        tbs.read_all(Error::BadDer, |tbs| {
+        tbs.read_all(Error::TrailingData("TBS"), |tbs| {
```

Comment:
Should this be prefixed with `Cert::` to match the other two instances above? 

Along similar lines, WDYT about exchanging "wrapper" for "Certificate" and "TBS" for "TBSCertificate" to better line up w/ 5280?

Reply:
I replaced the outer `Cert::wrapper` with just `Cert`, the inner one with `Cert::signed_data` and `TBS` with `Cert::tbs_certificate` (after getting ChatGPT to explain to me what "TBS" actually means"). WDYT?

Reply:
Sounds good to me. 

>  (after getting ChatGPT to explain to me what "TBS" actually means")

Did it do a good job? :robot: :speech_balloon:

Reply:
Not too bad!

<img width="657" alt="Screenshot 2023-08-01 at 20 30 20" src="https://github.com/rustls/webpki/assets/158471/a74cb4d8-c158-489f-ba22-e4e498f0a242">

It didn't have a great answer to this:

<img width="676" alt="Screenshot 2023-08-01 at 20 31 48" src="https://github.com/rustls/webpki/assets/158471/896ddd16-dfd5-46c4-bbeb-fedcefcaba44">

But once I knew what I was looking for I could find it in the RFC.